parsons/catalist/catalist.py

```diff
@@ -0,0 +1,388 @@
+"""Utilities for working with the Catalist Match API
+
+Install dependencies with `pip install parsons[catalist]`
+"""
+
+import base64
+import logging
+import os
+import tempfile
+import time
+import urllib
+from typing import Optional, Union, Dict, List
+from zipfile import ZipFile
+
+import requests
+from parsons.etl import Table
+from parsons.sftp import SFTP
+from parsons.utilities.api_connector import APIConnector
+
+logger = logging.getLogger(__name__)
+
+# Currently the default and only available template for working with the Match API
+template_id = 48827
```

Comment:
Makes sense! Do we anticipate Catalist making other templates available? If so might make sense to paramaterize this in the API call with the current value as a default, but I'll defer to you here

Reply:
Done with 08b48339a9721afe29c115f7b2fa47650ac3d852